.github/workflows/pythonapp.yml

```diff
@@ -22,6 +22,11 @@ jobs:
         django: [2.2, 3.1, 3.2]
     steps:
     - uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+    - name: 'fetch master'
```

Comment:
this looks weird, how is it related to darker?

Reply:
Because darker compares with the "master" branch. But the normal checkout will not make this branch available ;)